main.py

```diff
@@ -1,10 +1,27 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
+
+    for player_name, player_info in players.items():
+        print(player_info)
+        race, create = Race.objects.get_or_create(name=player_info["race"]["name"],
```

Comment:
The variable 'create' is used here, but it is not utilized later in the code. Consider renaming it to 'created' to better reflect its purpose, or remove it if it's not needed.